cmd/codegen/main.go

```diff
@@ -0,0 +1,134 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"go/format"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/getkin/kin-openapi/openapi3"
+)
+
+var buffer = bytes.NewBufferString("")
```

Comment:
It’s better not to create variables at the package level, so you can create a buffer in the main function and pass it as arguments to other functions